src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.html

```diff
@@ -0,0 +1,37 @@
+<div *ngIf="ready === true; then readyBlock else loadingBlock" id="load"></div>
+
+<ng-template #loadingBlock>
+    <app-loading-spinner></app-loading-spinner>
+</ng-template>
+
+<ng-template #readyBlock>
+    <div class="dialog-card">
+        <div class="title">Rewards History</div>
+        <div class="headers">
+            <div class="header-field"></div>
+            <div class="header-field">Vault</div>
+            <div class="header-field">Reward</div>
+            <div class="header-field">APY</div>
+            <div class="header-field">Weekly APY</div>
+            <div class="header-field">Date</div>
+        </div>
+        <div class="scroller">
+            <div class="values" *ngFor="let reward of rewards">
+                <div class="value-field"></div>
+                <div class="value-field">{{reward.vault}}</div>
+                <div class="value-field">{{reward.reward | number: '1.8-8'}}</div>
+                <div class="value-field">{{reward.apy | number: '1.2-2'}}%</div>
```

Comment:
hide APY, in this DTO it was deprecated
add a link to etherscan
use hash from ID (it is hash_logNumber)

Reply:
👍